app/models/domain_block.rb

```diff
@@ -12,11 +12,12 @@
 #
 
 class DomainBlock < ApplicationRecord
-  enum severity: [:silence, :suspend]
+  enum severity: [:silence, :suspend, :media_only]
```

Comment:
That's reasonable. Should the menu option still be called "Reject media only" in the frontend?